doc/conf.py

```diff
@@ -7,23 +7,30 @@
 from pathlib import Path
```

Comment:
File `doc/_static/.gitempty` is not required, since there are other files in that subdir.

Reply:
Fixed in e29730d7.